pkg/commands/process/settings/rules/javascript/express/cross_site_scripting/testdata/no_xss.js

```diff
@@ -1,6 +1,24 @@
 const express = require("express");
 const app = express();
 
-app.get("/goos", (_, res) => {
-  res.send("<p>hello world</p>")
+app.get("/good", (_, res) => {
+  return res.send("<p>hello world</p>")
+})
+
+app.get("/good-2", (req, res) => {
+  return res.send(JSON.stringify({
+    view: goodView,
+    success: true,
+    text: 'Good practice'
+  }))
+})
+
+app.get("/good-3", () => {
+  const results = newResult(search);
+
+  res.send(results);
+})
+
+app.get("/good-4", () => {
+  return res.send({ success: false, text: `User ${req.params.user_id} not found` });
```

Comment:
I guess it could also be `req.params["user_id"]` which wouldn't be found as it it. In which conditions did you see it fail before / raise false positives?

Reply:
True, it won't catch `req.params["user_id"]`. Will add some test cases for that one. 

This fix is more for the false positives that have come out of the batte testing: https://www.notion.so/bearer/Curio-battle-test-review-0-25-Javascript-55dcf15da8ff49088218ee852f7498bf

Reply:
```javascript
res.send({
  body: inspect(req.body, { depth: Infinity, maxStringLength: 100 })
})
```

We were discussing with @didroe about a way to mark a detection as `not-risky`. I guess this is another instance of it.
🤔

Reply:
Resolving so PR can be merged :)